### PR TITLE
[main] Merge xcode12.3 into main

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: Make.versions [hunk 1]
```diff
@@ -66,8 +66,7 @@ MAC_PACKAGE_VERSION=7.3.0.$(MAC_COMMIT_DISTANCE)
 # WARNING: Do **not** use versions higher than the available Xcode SDK or else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
 # When bumping the major macOS version in MACOS_NUGET_VERSION also update the macOS version where we execute on bots in jenkins/Jenkinsfile (in the 'node' element)
 
-IOS_NUGET_VERSION=14.2.100
-TVOS_NUGET_VERSION=14.2.100
-WATCHOS_NUGET_VERSION=7.1.100
-MACOS_NUGET_VERSION=11.0.100
-MACCATALYST_NUGET_VERSION=14.2.100
```
Comment (rolfbjarne) [inline Make.versions:73]: MACCATALYST_NUGET_VERSION disappeared!

File: src/frameworks.sources [hunk 2]
```diff
@@ -1952,6 +1953,7 @@ IOS_FRAMEWORKS =         \
 	$(COMMON_FRAMEWORKS) \
 	AVKit \
 	Accounts \
+	AdServices \
```
Comment (rolfbjarne) [inline src/frameworks.sources:1956]: Does `AdServices` exist for Mac Catalyst as well?

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: Make.versions (hunk 1 above)
Comment (dalexsoto) [reply Make.versions:73]: whoops, nice catch

File: src/frameworks.sources (hunk 2 above)
Comment (dalexsoto) [reply src/frameworks.sources:1956]: Yes according to docs [Mac Catalyst 14.3+](https://developer.apple.com/documentation/AdServices?language=objc)

File: Make.config [hunk 3]
```diff
@@ -36,7 +36,7 @@ $(TOP)/Make.config.inc: $(TOP)/Make.config $(TOP)/mk/mono.mk
 
 include $(TOP)/Make.versions
 
-APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-8/e30c41de32f64eca655953b17c78c1d9d70249e3/39/package/bundle.zip
+APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-8/87a1b18d8890ab76c4c658abd331dfde68d630ab/49/package/bundle.zip
```
Comment (spouliot) [inline Make.config:39]: we might want to compare with the inserted package for xcode12.3
or at least make a note, somewhere, to do it later - once published

**Comment by dalexsoto** [comment]
@rolfbjarne Fixed, thanks!

**Review by spouliot — APPROVED** [review]

File: src/frameworks.sources (hunk 2 above)
Comment (rolfbjarne) [reply src/frameworks.sources:1956]: Then you need an entry in the list of Mac Catalyst frameworks in this file (`MACCATALYST_FRAMEWORKS`).

Comment (dalexsoto) [reply src/frameworks.sources:1956]: @rolfbjarne done, thank you!

**Review by rolfbjarne — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by rolfbjarne** [comment]
build

**Comment by dalexsoto** [comment]
Merging since everything else is green, will make another PR to verify the API diff, it looks like we hit an unforeseen case for the API diff https://github.com/xamarin/xamarin-macios/pull/10264/files#diff-23232b8bb6d3c19948ebd0a6a98bb7278ff2b7db4f145460e8a1e65ff5f3d602R171 triggered by the change of Xcode, @rolfbjarne can you have a look? Maybe we need a way to ignore the check or account for it.

**Comment by rolfbjarne** [comment]
> @rolfbjarne can you have a look? Maybe we need a way to ignore the check or account for it.

It's a foreseen case, just not a supported one :smile: It shouldn't happen all that often, and if it turns out to be a problem we can fix it. Although it probably shouldn't be an error, it could be just a warning instead. I'll see if I can change that at some point.

**Pull request merged**